pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -52,8 +52,8 @@ def test_earth_relief_01d_gebco(data_source):
     assert data.gmt.registration == 0
```

Comment:
Metadata checks for gebco/gebcosi seems to have been left out from #1818
```suggestion
    assert data.gmt.registration == 0
    assert data.attrs["units"] == "meters"
    assert data.attrs["long_name"] == "Earth elevation relative to the geoid"
    assert data.attrs["vertical_datum"] == "EGM96"
    assert data.attrs["horizontal_datum"] == "WGS84"
```

Reply:
Done at 74fa646130fb1586d828abc8197b26ab4a6a0213.